tests/__init__.py

```diff
@@ -802,6 +802,10 @@ class SnapshotFailed(Exception):
     pass
 
 
+class SnapshotUnexpectedError(Exception):
```

Comment:
Don't see this used

Reply:
oops! had gone with one approach and then switched it up